### PR TITLE
DI-7 - add card 7

### PR DESCRIPTION
**Pull request by denishukov**
# Card 7
- added card 7;
- fixed styles;
- move all queries to the bottom of the page.
## Screenshorts
desktop
Screen Shot 2022-10-14 at 8 13 16 PM

mobile
Screen Shot 2022-10-14 at 8 13 32 PM

### CODE REVIEW

**Comment by elmirakim** [comment]
Thank you for such a detailed approach to the project. It looks amazing.

**Comment by elmirakim** [comment]
Elon Musk was a good one ^^

File: index.html [hunk 1]
```diff
@@ -6,43 +6,47 @@
...
+                <a class="home-link" href="#"><i class="fa-solid fa-house"></i><span>Seytech</span></a>
+                <ul class="main-nav">
+                    <li><a href="#"><i class="my-icon fa-solid fa-earth-americas" title="news"></i></a>
+                    </li>
+                    <li><a href="#"><i class="my-icon fa-solid fa-user" title="user account settings"></i></a></li>
+                    <li><a href="#"><i class="my-icon fa-solid fa-envelope" title="messages"></i></a></li>
+                </ul>
+            </div>
+        </header>
+        <!-- Ermek end -->
```
Comment (denishukov) [inline index.html:31]: Just prettier

File: style.css [hunk 2]
```diff
@@ -4,103 +4,105 @@
...
+.main {
+  display: grid;
+  grid-template-columns: auto;
+  max-width: 1400px;
+  margin: auto;
+}
+
 header {
-  background: skyblue;
+  background: var(--blue);
```
Comment (denishukov) [inline style.css:43]: we agreed to use our custom colors from the root

File: style.css [hunk 3]
```diff
@@ -202,350 +205,406 @@ hr {
...
+}
+
+button:hover {
+  color: black;
+  background-color: #888888;
+  cursor: pointer;
+
+}
+
+/* Denis - combine card 7 with card 8 */
```
Comment (denishukov) [inline style.css:294]: prettier formatter

**Pull request merged**